src/api/TKMember.m

```diff
@@ -664,6 +664,31 @@ - (void)getBalance:(NSString *)accountId
      }];
 }
 
+- (void)getBalances:(NSArray<NSString *> *)accountIds
+            withKey:(Key_Level)keyLevel
+          onSuccess:(OnSuccessWithTKBalances)onSuccess
+            onError:(OnError)onError {
+    [client
+     getBalances:accountIds
+     withKey:keyLevel
+     onSuccess:onSuccess
+     onError:^(NSError *error){
+         if ([error.domain isEqualToString:kTokenRequestErrorDomain]
+             && error.code == RequestStatus_MoreSignaturesNeeded
+             && keyLevel == Key_Level_Low) {
+             // Request again with Key_Level_Standard if more signatures are needed
+             [client
+              getBalances:accountIds
+              withKey:Key_Level_Standard
```

Comment:
asking, not saying: is "Standard" enough? I ?think? I heard someone say something about some accounts needing SCA for getBalance sometimes. I ?think? "SCA" on objc means "PRIVILEGED". (Hmm, I see a couple of "?think?"s in what I just wrote, so I'm probably wrong. But it would be good for me to learn what's correct.)

Reply:
In the sdk design, only Low key can avoid the SCA. Maybe privilege keys are "privilege" somewhere else but not in SCA scenario.

Reply:
Aha, makes sense. thank you